src/bundle/Resources/public/scss/_knp-menu.scss

```diff
@@ -0,0 +1,37 @@
+.ibexa-knp-menu {
```

Comment:
This should not be named `knp-menu`. The backend implementation may change and we have multiple knp menus on one page and each will look differently.